changelog.md

```diff
@@ -7,6 +7,7 @@
 * [3332](https://github.com/zeta-chain/node/pull/3332) - implement orchestrator V2. Move BTC observer-signer to V2
 * [3360](https://github.com/zeta-chain/node/pull/3360) - update protocol contract imports using consolidated path
 * [3349](https://github.com/zeta-chain/node/pull/3349) - implement new bitcoin rpc in zetaclient with improved performance and observability
+* [3381](https://github.com/zeta-chain/node/pull/3381) - some code refactoring in zetaclient for Bitcoin RBF adoption
```

Comment:
> some code refactoring

this is not a way to describe a change, should be descriptive

Reply:
updated: https://github.com/zeta-chain/node/pull/3381/commits/37f4ebbd5ed1c86d7ff7be5912c7c0cf7212a0d6